model/src/cfd.rs

```diff
@@ -2318,21 +2318,26 @@ impl Dlc {
     pub fn signed_cet(
         &self,
         attestation: &olivia::Attestation,
-    ) -> Result<Result<Transaction, IrrelevantAttestation>> {
-        let cets = match self.cets.get(&attestation.id) {
+    ) -> Result<Result<Result<Transaction, IrrelevantAttestation>, PriceOutOfRange>> {
```

Comment:
It's not strictly necessary to model it as separate results. It could be modelled as an enum too.

The `PriceOutOfRange` error needs to be handled separately because when we introduce liquidation events we need to ensure that we don't emit an error if the attested price is out of range for a liquididation event (see https://github.com/itchysats/itchysats/commit/151972547b99336f2709b6803cc3345cc211f6f2#diff-f078e5601c18fcb29f773d04d16902354c8f66a57b41aedaea122e11a6f8fd3dR1397-R1402).

Reply:
Ah, now I get it. Thanks.

Reply:
Maybe a solution involving an `enum` would actually be nicer. A tripple `Result` is not particularly beautiful. Not a blocker though.